server/map_rpc_server.go

```diff
@@ -252,6 +272,10 @@ func (t *TrillianMapServer) SetLeaves(ctx context.Context, req *trillian.SetMapL
 			}
 		}
 
+		if t.opts.UseSingleTransaction {
+			glog.Infof("EXP: rolled %d transactions up into single commit", txRolledUp)
```

Comment:
Info for every `SetLeaves` is going to be voluminous, but maybe OK for now.

Reply:
Moved to V(1)